frontend/src/app/versions.tsx

```diff
@@ -116,14 +119,23 @@ class Versions extends Component<IProps, any> {
 
         return (
             <React.Fragment>
-                {this.state.successAlertVisble && <Alert
+                {this.state.successAlertVisible && <Alert
                     variant="success"
                     title="Edit Metadata"
                     action={<AlertActionCloseButton onClose={this.hideSuccessAlert} />}
                 >
                     Update Successful!
           </Alert>
                 }
+
+                {this.state.publishAlertVisible && this.state.canChangePublishState === false && <Alert
```

Comment:
Why is `canChangePublishState` also in this conditional? Why isn't `publishAlertVisible` sufficient by itself?

Reply:
canChangePublishState is introduced in this feature branch as a flag that checks if metadata is set before a version can be published

Reply:
Ok, I understand the function of the property, I'm just not sure why it appears in this specific place in the code. Would we ever run into a scenario where `publishAlertVisible` was true and `canChangePublishState` was also true? If not then why not simplify the code and remove `canChangePublishState` from the conditional?

Reply:
So are you saying that the behavior of the application DOES change if we remove THIS particular check for `canChangePublishState`?

Reply:
Yes, it stops working.

Reply:
So I looked into this myself and, lo and behold, removing this check DOES change the behavior of the application!

...but, it's because of a bug elsewhere in the code (a small one). :)

Rather than try and describe it here, I just fixed it in my own branch. I'll open a PR for it.